contributing_to_docs/create_or_edit_content.adoc

```diff
@@ -9,21 +9,29 @@
 toc::[]
 
 == Before you begin
-Before you proceed, we recommend that you have completed the following:
+Before you create or edit content:
 
-* Read and reviewed the link:contributing.adoc[Contribute to OpenShift
-documentation] topic to understand some basics
-* link:tools_and_setup.adoc[Installed and set up the tools and software]
+* Read and review the link:contributing.adoc[Contribute to OpenShift documentation]
+topic to understand some basics
+* link:tools_and_setup.adoc[Install and set up the tools and software]
 required to contribute
-* Read and reviewed the link:doc_guidelines.adoc[Documentation guidelines] topic
+* Read and review the link:doc_guidelines.adoc[Documentation guidelines] topic
 to understand the basic guidelines for consistency
 
 == Ensure your local repository is in sync with the remote
 Before you create a local working branch, it is good practice to ensure that
-your local `master` branch is in sync with the remote and that you have all the
+your local source branch is in sync with the remote and that you have all the
 latest changes. You must also ensure that your forked repository is also in sync
 with the remote repository.
 
+[NOTE]
+====
+Because most changes in this repository must be committed to the `master`
+branch, the following process always uses `master` as the name of the source
+branch. If you need to use another branch as the source for your change, make
+sure that you consistently use that branch name instead of `master`
+====
+
```

Comment:
This note is not relevant for 4.0 but makes sense once 4.0 is launched and we switch master to it. I am in favor of keeping it.